src/main/java/uk/nhs/tis/trainee/usermanagement/api/UserAccountResource.java

```diff
@@ -110,4 +110,22 @@ ResponseEntity<Void> resetUserAccountMfa(@PathVariable String username) {
     log.info("MFA reset for user '{}'.", username);
     return ResponseEntity.noContent().build();
   }
+
+  /**
+   * Delete TSS Cognito account for the given user.
+   *
+   * @param username The username of the user.
+   * @return 204 No Content, if successful.
+   */
+  @PostMapping("/delete-account/{username}")
```

Comment:
Would a `DELETE` method be clearer than `POST`?
We would need to update the security config too.